### DISCUSSION TITLE
open the `showLastUpdateTime `, all docs show `Last updated on 10/14/2018`

### DISCUSSION THREAD

**Discussion by 668168** [facebook/docusaurus#7229] [Q&A]
open the showLastUpdateTime option: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#showLastUpdateTime
all page show the same time:

```
Last updated on 10/14/2018
(Simulated during dev for better perf)
```

backgroud: 
i do not use git , just npm run build  then  deploy to my  VPS nginx     .

**Comment by slorber** [comment]
> `(Simulated during dev for better perf)`

This message is only displayed in dev mode, not when using `docusaurus build` or `npm run build`

I don't know what I'm supposed to see on the link above 🤷‍♂️ 

<img width="371" alt="CleanShot 2022-04-22 at 17 46 19@2x" src="https://user-images.githubusercontent.com/749374/164749227-eb1ee04d-db27-4ad1-985f-797525860c6a.png">

If you have a problem on your site, please give a link to your site instead

**Reply by 668168** [reply]
i reinstall the VPS os, then reinstall the docusaurus, just now. 
only add  `showLastUpdateTime: true, `

the result with the blow cmd show at  193.123.235.145:3000, show "Last updated on 10/14/2018" 
`npm run start -- --host 0.0.0.0`

the result with the cmd blew , deploy with nginx , show at test.88.uy, not show any date info
`npm run build `

**Comment by 668168** [comment]
hi, @slorber , could you see the webpage?

**Reply by slorber** [reply]
@shuivin I am unable to help you because the problem and instructions are very unclear to me. I won't invest my time unless the instructions are clear

**Reply by 668168** [reply]
hi, Slorber,
thanks for your relay patiently . sorry for my  pool English , may be I mis-understand something you want to me to supply.

you said you need "If you have a problem on your site, please give a link to your site instead" 5 days ago.

so I show you the link 5 days ago.  for shortly:
https://test.88.uy                        ### builded web site deploy with nginx.
https://193.123.235.145:3000   ### for  just for npm run start

any thing i has been lost?

**Reply by slorber** [reply]
I have no idea what I'm supposed to do with these 2 links. 
And the IP link seems mostly offline for me (worked yesterday but not stable).

Please give me step by step instructions, for each link, what I'm supposed to see there and why it's a problem. Throwing me 2 links without explanation doesn't help.

> [test.88.uy](https://test.88.uy/) ### builded web site deploy with nginx.

I don't see "last updated" here

> [193.123.235.145:3000](https://193.123.235.145:3000/) ### for just for npm run start

You are not supposed to use "npm run start" to deploy a site. No matter what I see here, it shouldn't be relevant because you shouldn't do that in the first place

**Reply by 668168** [reply]
Thanks for your patient to help me,  slorber, 

the IP link just work without ssl with chrome.  you can visit it by http://193.123.235.145:3000 

# introduction  the issue step by step:

## STEP1:  install docusaurus with blew command ;  only add `showLastUpdateTime: true,`; 

 ```
npx create-docusaurus@latest website classic 
cd website
```
then , just edit the config file , add line `showLastUpdateTime: true,`
<img width="490" alt="image" src="https://user-images.githubusercontent.com/1046701/165788960-fa25d74f-0d9e-4541-8380-08ed55dc8bf5.png">


## STEP2: Build and Deply.

`yarn build --out-dir /home/wwwroot/test.88.uy`
note: the path is the nginx wwwroot path.
then you can visit the intro page by https://test.88.uy/docs/intro
but there is not sow the "Last updated on ... " at right bottom of  the page, just blank.
<img width="993" alt="image" src="https://user-images.githubusercontent.com/1046701/165789737-c800ba75-cb9d-400f-9712-09949b88cdb0.png">


##  STEP3: run ` npm run start -- --host 0.0.0.0`  command

run the blew command to visit the sam intro page.
` http://193.123.235.145:3000/docs/intro `
show "Last updated on 10/14/2018 (Simulated during dev for better perf) "
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/1046701/165790598-bedf95de-a155-4b8d-9035-1b53eb8e2672.png">

the system date is normal.
<img width="331" alt="image" src="https://user-images.githubusercontent.com/1046701/165790720-d4a301a8-2458-4bfc-a7e9-54777641406b.png">

#  any thing I do not  describe clearly , just let me know.

**Reply by Josh-Cena** [reply]
@shuivin

1. `npm run start` starts a server in development mode. The text there literally says why it's "10/14/2018": because it's simulated in development mode, because `git log` is an expensive action.
2. `npm run build` builds the app in production mode, where it's supposed to actually gather git history. However, your deployment environment does not have git, so it outputs nothing. If you check the build log, you should find warnings like "can't find git" or "can't retrieve history".

**Reply by 668168** [reply]
Yes!
I notice the git warning log, and agree with your appoint "actually gather git history. "
But confused by the  different phenomena with `npm run start` last week. 
You explain make me clearly.

# Conclusion

**must deploy with git  if someone like me want to show the "last update time ..."**

# Advice

show the update time by a mete tag such as "modifyTime: 20220501" maybe another choice.

Thanks for @Josh-Cena explain cleanly and quickly .
Thanks for @slorber' kindly, nice, and patiently for put up with my unorganized instructions.

**Reply by slorber** [reply]
Thanks @shuivin , created an issue here with your proposal: https://github.com/facebook/docusaurus/issues/7263
